edge_computing/image-base-install/ibi-edge-image-based-install-standalone.adoc

```diff
@@ -10,12 +10,12 @@ You can manually generate a configuration ISO by using the {product-title} insta
 
 include::modules/ibi-create-standalone-config-iso.adoc[leveloffset=+1]
 
-[role="_additional-resources"]
-.Additional resources
-
-* xref:../../openshift_images/managing_images/using-image-pull-secrets.adoc[[Using image pull secrets]
-
 include::modules/ibi-installer-configuration-config.adoc[leveloffset=+2]
 
 include::modules/ibi-extra-manifests-standalone.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
+== Additional resources
+
+* link:https://docs.openshift.com/container-platform/4.15/openshift_images/managing_images/using-image-pull-secrets.html[Using image pull secrets]
```

Comment:
@eranco74 Do you know if there's a reason for using the 4.15 version of this link?